content/en/docs/tasks/observability/distributed-tracing/configurability/telemetry-api/index.md

```diff
@@ -0,0 +1,183 @@
+Typical configuration includes service address and port for the provider, as well as establishing
+a limit on max tag length supported by the provider.
+
+Each configured provider *must* be named. That name will be used to refer to the provider in the
+Telemetry API.
+
+An example set of provider configuration in `MeshConfig` is:
+
+{{< text yaml >}}
+data:
+    mesh: |-
+       # The following content defines two example tracing providers.
+        extensionProviders:
+        - name: "zippy-kin"
+          zipkin:
+            service: "zipkin.istio-system.svc.cluster.local"
+            port: 9411
+            maxTagLength: 56
+        - name: "stacky-driver"
+          stackdriver:
+            maxTagLength: 256
+        # A mesh default for provider can be specified to simplify Telemetry resources
+        defaultProviders:
+            tracing: "stacky-driver"
+{{</ text >}}
+
+### Configuring mesh-wide tracing behavior
+
+Telemetry API resources inherit from the root configuration namespace for a mesh, typically `istio-system`. To configure
+mesh-wide behavior, add a new (or edit the existing) `Telemetry` resource in the root configuration namespace.
+
+Here is an example configuration that uses the provider configuration from the prior section:
+
+{{< text yaml >}}
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: mesh-default
+  namespace: istio-system
+spec:
+  tracing:
+  - providers: # only a single tracing provider can be selected at this time
+    - name: zippy-kin
+    customTags:
+      foo:
+        literal:
+          value: bar
+    randomSamplingPercentage: 100
+{{</ text >}}
+
+This configuration overrides the default provider from `MeshConfig`, setting the mesh default to be the `"zippy-kin"`
+provider. It also sets the mesh-wide sampling percentage to be `100`, and configures a tag to be added to all trace
+spans with a name of `foo` and a value of `bar`.
+
+### Configuring namespace-scoped tracing behavior
+
+To tailor the tracing behavior for individual namespaces, add a `Telemetry` resource to the desired namespace. Any tracing
+fields specified in the namespace resource will completely override the inherited field configuration from the configuration hierarchy.
+
+For example:
+
+{{< text yaml >}}
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: namespace-override
+  namespace: myapp
+spec:
+  tracing:
+  - customTags:
+      userId:
+        header:
+          name: userId
+          defaultValue: unknown
+{{</ text >}}
+
+When deployed with into a mesh with the prior mesh-wide example configuration, this will result in
+tracing behavior in the `myapp` namespace that sends trace spans to the `zippy-kin` provider and
+randomly selects requests for tracing at a `100%` rate, but that sets custom tags for each span with
+a name of `userId` and a value taken from the `userId` request header. The custom tags behavior completely
```

Comment:
Clarify: myapp ns will not get foo:bar tag and instead only get the header tag.

Reply:
added more text here.